doc/openapi-apicurio.md

```diff
@@ -0,0 +1,79 @@
+## Usin APICurio to edit an OpenAPI spec and visualise Kuadrant extensions
+
+This guide assumes you have APICurio already running.
+See https://www.apicur.io/ for info on how to install APICurio Studio.
+
+### Editing the spec
+
+Open or import your OpenAPI spec in the APICurio UI.
+You can modify the source of the spec from the UI.
+There are a few different configuration and extension points supported by APICurio, and also supported by the `kuadrantctl` cli.
+
+To generate a HTTPRoute for the API, add the following `x-kuadrant` block to your spec, replacing values to match your APIs details and the location of your Gateway.
+
+```yaml
+info:
+    x-kuadrant:
+        route:
+            name: petstore
+            namespace: petstore
+            labels:
+                deployment: petstore
+                owner: jbloggs
+            hostnames:
+                - 'petstore.example.com'
+            parentRefs:
+                -   name: prod-web
+                    namespace: kuadrant-multi-cluster-gateways
+                    kind: Gateway
+```
```

Comment:
```suggestion
```yaml
info:
    x-kuadrant:
        route:
            name: petstore
            namespace: petstore
            hostnames:
                - 'petstore.example.com'
            parentRefs:
                -   name: prod-web
                    namespace: kuadrant-multi-cluster-gateways
                    kind: Gateway
```
```

Reply:
can probably remove the labelling stuff